docs/17-common-problems-and-their-solutions.md

```diff
@@ -4,97 +4,22 @@ Common problems and their solutions
 These instructions will assume you start in the project root directory and Sleeper has been built
 (see [the developer guide](12-dev-guide.md) for how to set that up).
 
-## EOFException when using client classes
-
-An exception of the following form might be due to a change in schema:
-
-```bash
-Caused by: java.io.EOFException
-     at java.io.DataInputStream.readFully(DataInputStream.java:197)
-     at java.io.DataInputStream.readUTF(DataInputStream.java:609)
-     at java.io.DataInputStream.readUTF(DataInputStream.java:564)
-Exception in thread "main" sleeper.core.statestore.StateStoreException: Exception querying DynamoDB
-```
-
-When cdk deploy is used to deploy a table, the state store is initialised. This initialisation writes the initial
-partitions to the partitions part of the state store (if using the default DynamoDB state store then this will be a
-table named `sleeper-<instance-id>-table-<table-name>-partitions`). The partitions are specific to the schema of the
-table (specifically to the row keys specified). Therefore if you deploy a table with a certain schema, and then attempt
-to change the schema by updating the table properties file, any interactions with the partitions table will not be able
-to read the partitions.
-
-If you want to reinitialise a table with a new schema, see the section below on reinitialising a table.
-
 ## Out of memory error from standard ingest tasks
 
 If standard ingest tasks fail with an out of memory error ("Exception in thread main java.lang.OutOfMemoryError: Java
 heap space") then this is likely due to the tasks not being able to store the specified number of records in memory.
-Standard ingest works by reading a certain number of records (given by `sleeper.ingest.memory.max.batch.size`) into
+Standard ingest works by reading a certain number of records (given by `sleeper.ingest.arrow.working.buffer.bytes`) into
 memory. These are sorted and then written to a local file. This process is repeated a certain number of times until a
-certain number of records in total (given by `sleeper.ingest.max.local.records`) have been written to local disk.
-Sensible values for the parameters `sleeper.ingest.memory.max.batch.size` and `sleeper.ingest.max.local.records`
+certain number of records in total (given by `sleeper.ingest.arrow.max.single.write.to.file.records`) have been written to local disk.
+Sensible values for the parameters `sleeper.ingest.arrow.max.local.store.bytes` and `sleeper.ingest.arrow.batch.buffer.bytes`
```

Comment:
These properties are different from the ones above, whereas previously it was referring to the same properties. It looks like this needs a bit more explanation?